src/patternfly/components/ContextSelector/context-selector-toggle.hbs

```diff
@@ -1,4 +1,4 @@
-<button class="pf-c-context-selector__toggle{{#if context-selector-toggle--modifier}} {{context-selector-toggle--modifier}}{{/if}}"
+<button class="pf-c-context-selector__toggle{{#if context-selector-toggle--IsPlain}} pf-m-plain{{/if}}{{#if context-selector-toggle--modifier}} {{context-selector-toggle--modifier}}{{/if}}"
```

Comment:
You have some context selector changes in this PR. It looks like maybe you created this branch from the context selector branch you were working on? You can see this branch has 2 commits - one for the context selector, and one for the notification drawer work - https://github.com/patternfly/patternfly/pull/4117/commits

Reply:
Fixed in https://github.com/patternfly/patternfly/pull/4117/commits/93f0792c68c556a4743398c3e0538e53de51bf8b